RELEASE.md

```diff
@@ -68,11 +50,7 @@
     set +e
     ```
 
-* push to local github
-* merge (this step and the above one can be skipped if you just do it on the the repo's release branch itself)
-* git tag (your version number)
-* git push --tags
-* get a clean release (as in, fresh clone)
-* npm publish
+* `npm publish`
+* `git push --follow-tags`
```

Comment:
I originally had it that way, but then I added in the `postpublish` hook to clean up `dist/`, so I swapped the command order so the cleanup commit would be pushed as well. I could change it back, then add another push after publish, though:

```bash
git push --follow-tags
npm publish
git push
```